backend/src/models/user.model.ts

```diff
@@ -11,12 +11,13 @@ export interface IUser extends Document {
   image?: string;
   socials?: [{ link: string; social: ISocial['_id'] }];
   active: boolean;
+  section: boolean;
 }
 
 const userSchema = new Schema({
   name: String,
   title: String,
-  email: { type: String, required: true, unique: true },
+  email: { type: String, unique: true },
```

Comment:
Could you tell us, why email was changed into optional?

Reply:
Because you told me that is problem with this field when is required. Maybe I didn't understand you correctly on Slack

Reply:
Did you check what is the difference between required: true and without this property? 😄 

In the frontend side of this task, we can see https://github.com/Geeks-Academy/structure/issues/94#issue-884777467 that in case we have selected `section` - email and title should be populated base on name